doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2291,26 +2240,19 @@ the ``Generalizable`` vernacular command to avoid unexpected
 generalizations when mistyping identifiers. There are several commands
 that specify which variables should be generalizable.
 
-.. cmd:: Generalizable All Variables
+.. cmd:: Generalizable {| All Variables | No Variables | {| Variable | Variables } {+ @ident } }
 
-   All variables are candidate for
+   All variables are candidates for
```

Comment:
This is the description for the `All Variables` variant. This should be made explicit.

Reply:
Check what I wrote.  I have no idea what a "generalization delimiter" is.

Reply:
This likely refers to the ``` `( @ident : @type ) ``` / ``` `{ @ident : @type } ``` / ``` `[ @ident : @type ] ``` from `binder` (I simplified the syntax on purpose).

Reply:
BTW, I think we should move this part of the `binder` syntax into a new nonterminal `generalizing_binder` to be defined around here. To be clear I am talking about:

![2020-03-23-114051_468x245_scrot](https://user-images.githubusercontent.com/1108325/77308388-29acc100-6cfb-11ea-9575-17a2757d84d9.png)

And we should also move the two rules just above:

![2020-03-23-114151_472x64_scrot](https://user-images.githubusercontent.com/1108325/77308491-506af780-6cfb-11ea-959d-c110ad239fa5.png)

to a new nonterminal `implicit_binders` to be defined in the section on "Implicit Argument Binders".